### ISSUE TITLE
Error repeatedly logs

### ISSUE THREAD

**Issue by jasperfirecai2** [elytra/MatterLink#8]
Hi,

A friend and I have tried to set up a matterlink and bridge but we've currently run into this issue whenever we type something in minecraft:
```
[21:09:59] [Server thread/INFO] [STDERR]: [matterlink.api.MessageHandler:transmitMessage:161]:  at matterlink.api.MessageHandler.transmitMessage(MessageHandler.java:166)
[21:09:59] [Server thread/INFO] [STDERR]: [matterlink.api.MessageHandler:transmitMessage:161]:  at matterlink.api.MessageHandler.transmit(MessageHandler.java:119)
[21:09:59] [Server thread/INFO] [STDERR]: [matterlink.api.MessageHandler:transmitMessage:161]:  at matterlink.api.MessageHandler.stop(MessageHandler.java:61)
```

these three lines keep repeating themselves and time everyone out from the server. What other info do you need from me to help resolve this?
NikkyAI/MatterLinkAPI#1

**Comment by NikkyAI** [comment]
i think i found the issue and what caused it.. a recursion hapopened where it could not send messages so the message about the error triggered the next error...

is fixed in 68144ae51232e25104ea4703aefb8f2ad3401dda

**Comment by jasperfirecai2** [comment]
Cool
